spython/utils/misc.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
Could you please add some docstrings to this file to explain not just the functions and ScopedEnvVar, but also the parameters, and add the missing license header?

Also, I believe that the rest of the code uses three parens `'''` for docstrings instead of quotes `"""` and we should be consistent.